src/main/java/org/candlepin/insights/inventory/InventoryService.java

```diff
@@ -99,6 +122,8 @@ private CreateHostIn createHost(ConduitFacts conduitFacts) {
             rhsmFactMap.put("RH_PROD", conduitFacts.getRhProd());
         }
 
+        rhsmFactMap.put("syncTimestamp", syncTimestamp);
```

Comment:
Please update the fact name to be SYNC_TIMESTAMP (following the pattern of other fact names) as that is what rhsm-subscriptions is expecting.